File: Fines2Wpf/Values/Designacion.cs
using SqlOrganize;
using Utils;

namespace Fines2Wpf.Values
{
    class Designacion : EntityValues
    {
        public Designacion(Db _db, string _entity_name, string? _field_id) : base(_db, _entity_name, _field_id)
        {
        }

        public override string ToString()
        {
            var s = "";

            EntityValues? v = ValuesTree("persona");
            if(v != null)
            {
                s += v.GetOrNull("nombres")?.ToString() ?? "?";
                s += " ";
                s += v.GetOrNull("apellidos")?.ToString() ?? "?";
                s += " ";
                s += v.GetOrNull("telefono")?.ToString() ?? "?";
            } else
            {
                s = "?";
            }

            return s.Trim();
        }
    }
}

    

File: Model_/Designacion_.php
<?php

namespace Fines2;

use \Fines2\Designacion;

use SqlOrganize\Sql\Entity;
use Exception;
use DateTime;

class Designacion_ extends Designacion
{

    public function getLabel(): string
    {
        return ($this->persona_?->nombres ?? "?"). " " . ($this->persona_?->apellidos ?? "?") . " " . ($this->persona_?->telefono ?? "?");
    }
    

}

